src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -257,15 +223,8 @@ private function isPublicProperty(string $class, string $property): bool
     }
 
     /**
-     * Gets the accessor method.
```

Comment:
this should be kept. You cannot have a long description without the short one before it.